server/parameters/CoreConfigParser.java

```diff
@@ -376,26 +376,30 @@ private static class Debugger extends Compound<CoreConfig.Log.Debugger> {
             private static final String name = "debugger";
             private static final String description = "Debuggers that may be enabled at runtime.";
 
-            private static final Predefined<CoreConfig.Log.Debugger.Reasoner> reasoner =
-                    predefined("reasoner", "Configure reasoner debugger.", new Reasoner());
-            private static final Set<Predefined<?>> parsers = set(reasoner);
+            private static final Predefined<CoreConfig.Log.Debugger.ReasonerTracer> reasonerTracer =
+                    predefined("reasoner-tracer", "Configure reasoner tracer.", new ReasonerTracer());
+
+            private static final Predefined<CoreConfig.Log.Debugger.ReasonerPerfCounters> reasonerPerfCounters =
+                    predefined("reasoner-perfcounters", "Configure reasoner perfcounters.", new ReasonerPerfCounters());
```

Comment:
Use the full description string "Configure per-transaction reasoner performance counters."

Reply:
Doesn't that sounds like you're configuring it per transaction?

Reply:
Humm good point, maybe "Configure transaction-bound reasoner performance counters"?